src/ServiceProvider.php

```diff
@@ -92,4 +99,19 @@ protected function loadMigrations()
     {
         $this->loadMigrationsFrom(__DIR__.'/../resources/migrations');
     }
+
+    /**
+     * Loads our groups
+     *
+     * @return void
+     */
+    protected function loadGroups(Rollout $rollout, array $groups)
+    {
+        foreach ($groups as $group) {
+            $instance = new $group();
```

Comment:
I could be beneficial to resolve this from the IOC so that you can use DI in your group classes.